pay-api/migrations/versions/2023_10_26_7d5231f2b9ac_update_eft_enable_payment_accounts.py

```diff
@@ -0,0 +1,33 @@
+"""18099-eft allowed flag
+
+Revision ID: 7d5231f2b9ac
+Revises: 2ef58b39cafc
+Create Date: 2023-10-26 13:55:46.291450
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '7d5231f2b9ac'
+down_revision = '2ef58b39cafc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Update eft_enable in payment_accounts table."""
+    # eft_enable set to true when it has invoices paid with EFT/DIRECT_PAY payment method historically
+    op.execute("set statement_timeout=20000;")
+    conn = op.get_bind()
+    conn.execute('UPDATE payment_accounts \
```

Comment:
I don't think this is necessary, we want to enable specific accounts via notebook. It will also give us flexibility to enable a subset of accounts if we want and we want to send a welcome email when we enable it for accounts.

A new route/endpoint is beneficial as we can secure it differently than the existing payment account endpoints and have separate logic like the welcome email when we enable this flag.

Reply:
I removed this file, it gonna handle by ticket 18100